packages/webapp/components/TreasuryBalance.tsx

```diff
@@ -19,12 +19,40 @@ import { useGnarsV2TokenBalanceOf } from "utils/sdk"
 import { formatEther } from "viem"
 import { useBalance } from "wagmi"
 import { ShredIcon } from "./Icons"
-
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { set } from "lodash"
+import { abbreviatedBalance } from "utils/numberAbreviation"
 export const TreasuryBalance = () => {
   const { data: treasuryBalance } = useBalance({
     address: TREASURY_ADDRESS,
   })
 
+  const [usdcBalance, setUsdcBalance] = useState(0);
+
+
+  useEffect(() => {
+    const fetchUSDCBalance = async () => {
+      try {
+        const address = "0xa1B74d2280966A89AC7e0F3A8bc5f0867C776d98";
+        const apiKey = process.env.ETHERSCAN_API_KEY;
+
+        const response = await axios.get(`https://api.etherscan.io/api?module=account&action=tokenbalance&contractaddress=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&address=${address}&tag=latest&apikey=${apiKey}`);
```

Comment:
can just use `fetch` here. no need to add axios

Reply:
ok, will try ! thanks !

Reply:
Done